internal/cli/default_setter_opts.go

```diff
@@ -151,7 +151,7 @@ func (opts *DefaultSetterOpts) orgs(filter string) (results interface{}, err err
 	}
 	switch r := orgs.(type) {
 	case *atlasv2.PaginatedOrganization:
-		if *r.TotalCount == 0 {
+		if r.TotalCount == nil || *r.TotalCount == 0 {
 			return nil, errNoResults
 		}
 		if *r.TotalCount > resultsLimit {
```

Comment:
as we don't know yet how to reproduce this issue (might be backend related), should we have nil check for this if statement too?

Reply:
code should not reach this line if `TotalCount` is `nill` but enter the previous `if` 🤔 Am I missing something?

Reply:
you're right, misread the check above